src/index.scss

```diff
@@ -0,0 +1,231 @@
+@import './variables';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: $font-family-roboto;
+}
+
+header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $white;
+  padding: 0 50px;
+  height: 60px;
+
+  .header__logo {
+    img {
+      height: 40px;
+    }
+  }
+
+  .nav {
+    &__list {
+      list-style: none;
+      display: flex;
+      padding: 0;
+      margin: 0;
+    }
+
+    &__item {
+      position: relative;
+      margin-left: $spacing-medium;
+    }
+
+    &__link {
+      text-decoration: none;
+      color: $text-color-dark;
+      text-transform: uppercase;
+      font-weight: 500;
+      padding: 10px 0;
+
+      &::after {
+        content: '';
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: -5px;
+        height: 2px;
+        background-color: $primary-color;
+        display: none;
+      }
+
+      &:hover,
+      &.is-active {
+        transition: $hover-transition;
+        color: $primary-color;
+      }
+
+      &:hover::after,
+      .nav__item:first-child &::after {
+        transition: $hover-transition;
+        display: block;
+      }
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid $secondary-color;
+  border-radius: $border-radius;
+  transition: $hover-transition;
+
```

Comment:
Your `.card` stylization is not matching the [Mockup](https://www.figma.com/design/ojkArVazq7vsX0nbpn9CxZ/Moyo-%2F-Catalog-(ENG)?node-id=32249-354&node-type=frame&t=4mEm6lkjfGKDAE3l-0)

![image](https://github.com/user-attachments/assets/aac82799-a098-438a-98bc-b1bbab66ed90)

Take a look on your paddings and margins to match it.

Reply:
Also, you should use the `stars`.

Reply:
You're adding too much space between the cards, as well. Take a look at it

![image](https://github.com/user-attachments/assets/b9a71b3e-6afe-4f6c-91d6-0184a112fb02)